### PR TITLE
Personal review branch

### PR DESCRIPTION
**Pull request by AyomipoMajek**

### CODE REVIEW

**Review by AyomipoMajek — COMMENTED** [review]
Hi [@AyomipoMajek](https://github.com/AyomipoMajek),

Status: Changes Required ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Good Highlights✔
Good commit messages
Adhered to GitHub flow
Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# Personal-Review
```
Comment (AyomipoMajek) [inline README.md:1]: Kindly delete the #Personal-Review as you already added a proper # in the line that followed.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#navbar{
+  position:fixed;
+  top:0;
+}
```
Comment (AyomipoMajek) [inline style.css:4]: kindly add a one line space between your first and second CSS command so your code looks very neat.

**Review by AyomipoMajek — COMMENTED** [review]
Hi [@AyomipoMajek](https://github.com/AyomipoMajek),

Status: Changes Required ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Good Highlights✔
Good commit messages
Adhered to GitHub flow
Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by Olamarx — APPROVED** [review]
Hi @AyomipoMajek,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of GitHub review features ✔️ 
- Grammatically correct sentences 👍🏼 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**